### PR TITLE
Add AppOptics to the observability landscape

### PR DESCRIPTION
**Pull request by cce**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] Are you only including a `repo_url` if your project is 100% open source? If so, you need to pick the single best GitHub repository for your project, not a GitHub organization.
* [x] Is your project closed source or, if it is open source, does your project have at least 250 GitHub stars?
* [x] Have you picked the single best (existing) category for your project?
* [x] Does it follow the other guidelines from the [new entries](https://github.com/cncf/landscape#new-entries) section?
* [x] Have you included a URL for your SVG or added it to `hosted_logos` and referenced it there?
* [x] Does your logo clearly state the name of the project/product and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [x] Does your project/product name match the text on the logo?
* [x] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?
* [x] 2 minutes after opening the pull request, the CNCF-Bot will post the URL for your staging server. Have you confirmed that it looks good to you and then added a comment to the PR saying "LGTM"?

### CODE REVIEW

**Comment by cce** [comment]
LGTM. However following the instructions "Put the project in the single category where it best fits" is hard given the separate categories "Monitoring" and "Tracing" — AppOptics provides host and infrastructure monitoring, custom metrics, APM and distributed tracing in a single product. I chose "Monitoring" even though AppOptics [supports tracing standards](https://github.com/appoptics/appoptics-apm-go/#opentracing) such as the CNCF-endorsed OpenTracing.

**Comment by dankohn** [comment]
Thanks, and nice job fixing our null ticker bug. Note that Lightstep also puts themselves in monitoring, even though CEO Ben Siegelman founded OpenTracing. So, I think you're in the right spot.

https://landscape.cncf.io/grouping=landscape&landscape=monitoring&selected=light-step

File: tools/crunchbase.js [hunk 1]
```diff
@@ -147,7 +147,7 @@ export async function fetchCrunchbaseEntries({cache, preferCache}) {
       var meAndParents = [result.data].concat(parents);
       var firstWithTicker = _.find( meAndParents, (org) => !!org.properties.stock_symbol );
       var firstWithFunding = _.find( meAndParents, (org) => !!org.properties.total_funding_usd );
-      if (firstWithTicker || c.ticker) {
+      if (!(c.ticker === null) && (firstWithTicker || c.ticker)) {
```
Comment (ZeusTheTrueGod) [inline tools/crunchbase.js]: I don't get it:
1) ticker === null
This is converted to `if (firstWithTicker)` in both cases
2) ticker !== null, i.e. it is present.
It is converted to `if (firstWithTicker || c.ticker)` in both cases

Comment (cce) [reply tools/crunchbase.js]: This change helped to get around the error I was getting in https://app.netlify.com/sites/landscape/deploys/5af0b75d8df89449e3d0e8cd where I was setting `stock_ticker: null` in landscape.yaml but the build didn't recognize that meant it shouldn't use the ticker symbol from Crunchbase. I'm trying to skip this block if a null ticker is explicitly set.

**Comment by leecalcote** [comment]
Thanks for this merge, @dankohn.

**Pull request merged**